creation/interfaces-and-networks.md

```diff
@@ -692,3 +681,164 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
 > into VMI guest. Because of that, when configuring SR-IOV operator
 > policies, make sure you define a pool of VF resources that uses
 > `driver: vfio`.
+> **Note:** Further instructions will assume the aforementioned operator
+> was installed.
+
+Once the operator is deployed, an [SriovNetworkNodePolicy
+](https://github.com/openshift/sriov-network-operator#sriovnetworknodeconfigpolicy)
+must be provisioned, in which the list of SR-IOV devices to expose (with
+respective configurations) is defined.
+
+Please refer to the following `SriovNetworkNodePolicy` for an example:
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-1
+  namespace: sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  mtu: 9000
+  nicSelector:
+    pfNames:
+    - ens1f0
+  nodeSelector:
+    sriov: "true"
+  numVfs: 8
+  priority: 90
+  resourceName: sriov_net
+```
+
+The policy above will configure the `SR-IOV` device plugin, allowing the
+PF named `ens1f0` to be exposed in the SRIOV capable nodes as a resource named
+`sriov_net`.
+
+It will also render the `spec` for an associated `SriovNetworkNodeState` object,
+which looks like:
+
+```yaml
+apiVersion: v1
+items:
+- apiVersion: sriovnetwork.openshift.io/v1
+  kind: SriovNetworkNodeState
+  metadata:
+    name: sriov-worker
+    namespace: sriov-network-operator
+    ownerReferences:
+    - apiVersion: sriovnetwork.openshift.io/v1
+      blockOwnerDeletion: true
+      controller: true
+      kind: SriovNetworkNodePolicy
+      name: default
+  spec:
+    interfaces:
+    - mtu: 9000
+      name: ens1f0
+      numVfs: 8
+      pciAddress: 0000:65:00.0
+      vfGroups:
+      - deviceType: vfio-pci
+        resourceName: sriov_net
+        vfRange: 0-7
+  status:
+    interfaces:
+    - Vfs:
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.2
+        vendor: 15b3
+        vfID: 0
+        deviceID: "1015"
+      ...
+  syncStatus: Succeeded
+```
+
+#### Start an SR-IOV VM
+
+Once all the SR-IOV components are deployed, it is needed to indicate how to
+configure the SR-IOV network. Refer to the following
+`SriovNetwork` for an example:
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  annotations:
+    operator.sriovnetwork.openshift.io/last-network-namespace: default
+  finalizers:
+  - netattdef.finalizers.sriovnetwork.openshift.io
+  generation: 2
+  name: sriov-mlx
+  namespace: sriov-network-operator
+spec:
+  ipam: |
+    {}
+  networkNamespace: default
+  resourceName: sriov_net ??
```

Comment:
I think it is ok, even without the `??`

Reply:
Vestigial leftover of my own clumsiness; removed it.